tsfresh/examples/driftbif_simulation.py

```diff
@@ -27,7 +27,7 @@ class velocity(object):
     1
     >>> print(ds.deterministic) # Equilibrium velocity
     0.0015191090506254991
-    >>> v = ds.simulate(20000) # Simulate velocity time series with 20000 time steps being disturbed by Gaussian white noise
+    >>> v = ds.simulate(20000) # Simulate velocity. Time series with 20000 time steps being disturbed by Gaussian white noise
```

Comment:
it is a time series of velocity, maybe write "Simulated velocity as a time series with ..."